src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx

```diff
@@ -76,6 +102,16 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ states, a
                 onChange={(e) => name(e.target.value)}
               />
             </Grid>
+            <Grid item xs={12}>
+              <NERAutocomplete
```

Comment:
From what the mockup looks like, you probably want the project name and assign a team fields to be in the same row, you can accomplish this by altering the xs, md attributes of the grid item for both the autocomplete and the project name input. you can also limit the size of the autocomplete via those values as well.